src/signals/incident-management/containers/Dashboard/Dashboard.tsx

```diff
@@ -57,11 +58,13 @@ const Dashboard = () => {
           <AreaChart />
         </StyledRow>
       </Route>
-      <Route path={`${DASHBOARD_URL}/signalering`}>
```

Comment:
Why is the feature flag necessary here? This component is not rendered when the flag is set to false right? This way the page is not accessible when the feature flag is true.

Reply:
- [ ] will delete it